src/renderer/constants/apiUrls.ts

```diff
@@ -0,0 +1,41 @@
+import { Epoch, Slot, Shard, BLSPubkey, BeaconBlock, uint8, IndexedAttestation } from '@chainsafe/eth2.0-types';
+
+export const API_URL: String = `https://${process.env.HOSTNAME || 'localhost'}:${process.env.PORT || '3000'}`;
```

Comment:
create config file which will contain all configuration and default values if env is missing.
Also API_URL should be passed as param to api client as we will instantiate multiple api client with different url at the same time